modules/debts_api/app/models/debts_api/v0/form5655_submission.rb

```diff
@@ -136,6 +136,43 @@ def streamlined?
       public_metadata.dig('streamlined', 'value') == true
     end
 
+    def vba_debt_identifiers
+      return [] if metadata.blank?
+
+      parsed_metadata = JSON.parse(metadata)
+      debts = parsed_metadata['debts'] || []
+
+      debts.map do |debt|
+        "#{debt['deductionCode']}#{debt['originalAR'].to_i}"
+      end.compact
+    rescue JSON::ParserError
+      []
+    end
+
+    def vha_copay_identifiers
+      return [] if metadata.blank?
+
+      parsed_metadata = JSON.parse(metadata)
+      copays = parsed_metadata['copays'] || []
+
+      copays.pluck('id').compact
```

Comment:
Pluck should be only used for ActiveRecord relations, please change this to something similar like, `copays.map { |copay| copay['id'] }.compact`

Reply:
@RachalCassity Got this fix merged in, thanks for the feedback!